tests/test_discord_app.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from unittest.mock import MagicMock
+import sys
+import os
+
+# Add the project root directory to the Python path to find 'src'
+project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+if project_root not in sys.path:
+    sys.path.insert(0, project_root)
```

Comment:
This is a bit of a hack. The cleanest solution is to make the project `pip`-installable, e.g. by adding a `pyproject.toml`, but I didn't want to make such a big change without discussion.